components/gc-intro/gc-intro-block-en.html

```diff
@@ -0,0 +1,88 @@
+---
+title: Introduction block pattern
+description: Documentation and working example for the introduction block pattern
+language: en
+altLangPage: gc-intro-block-fr.html
+dateModified: 2023-08-09
+layout: no-container
+---
+<div class="container">
+	<h1 property="name" id="wb-cont" dir="ltr">{{ page.title }}</h1>
+
+	<span class="wb-prettify all-pre hide"></span>
+	<p>The introduction block pattern introduces the content of a landing page.</p>
+
+	<h2>On this page</h2>
+	<ul>
+		<!-- <li><a href="#inc-components">Included components</a></li> -->
+		<li><a href="#before-start">Before you start</a></li>
+		<li><a href="#accessibility-guidance">Accessibility guidance</a></li>
+		<li><a href="#variations">Variations</a></li>
+		<li><a href="#eval-report">Evaluation and report</a></li>
+		<li><a href="#technical-docs">Technical documentation</a></li>
+	</ul>
+
+	<h2 id="before-start">Before you start</h2>
+	<p>The introduction block pattern is a mandatory element for institutional landing pages and topic pages. It’s optional for other landing pages.</p>
+
+	<ul>
+		<li><a href="">How to set up the institutional landing page</a></li>
+		<li><a href="">Introduction block pattern</a></li>
+	</ul>
+
+	<h2 id="accessibility-guidance">Accessibility guidance</h2>
+	<p>List of accessibility guidances to be followed to ensure component is used in respect of the accessibility guideline and EN 301 549 standard (including WCAG 2.1 Level AA).</p>
+	<p>There is <span>no accessibility guidance</span> defined.</p>
+
+	<h2 id="variations">Variations</h2>
+	<p>The introduction block pattern consists of:</p>
+	<ul>
+		<li>title</li>
+		<li>short description</li>
+	</ul>
+	<p>It can also include:</p>
+	<ul>
+		<li>image</li>
+		<li>supertask button</li>
+	</ul>
+
+	<h3>Introduction block</h3>
+	<div class="bg-center bg-cover" data-bgimg-srcset="img/1x1.png 991w, img/ip-cover-image-1200x726.jpg 992w">
+		<div class="container">
+			<div class="row mrgn-bttm-md">
+				<div class="col-md-7">
+				<h1 property="name" id="wb-cont">[Institution name]</h1>
+					<p>Short description of the institution’s mandate.</p>
+					<a class="btn btn-call-to-action" href="#">Super task button [optional]</a>
+				</div>
+			</div>
+		</div>
+	</div>
```

Comment:
This do create an heading structure issue.

There is a few alternative like informing the user of that irregular page structure or simpler will be to demo it from it's own dedicate page.

I do recommend to have the demo on a separate page and keep the source code here.

Reply:
Do you have an example I can use for creating separate demo page? Wouldn't it have the same issue though of two h1s on the page because the demo page will still have an h1

Reply:
For that you will need to create a new .html file in the folder. I suggest to use the layout "no-container" for this example.

That page markup could look like this:
```
---
{
	"title": "GC intro block with super task",
	"language": "fr",
	"altLangPage": "intro-block-fr.html",
	"layout": "no-container",
	"dateModified": "2023-09-13"
}
---
<div class="bg-center bg-cover" data-bgimg-srcset="img/1x1.png 991w, img/ip-cover-image-1200x726.jpg 992w">
	<div class="container">
		<div class="row mrgn-bttm-md">
			<div class="col-md-7">
			<h1 property="name" id="wb-cont">[Institution name]</h1>
				<p>Short description of the institution’s mandate.</p>
				<p><a class="btn btn-call-to-action" href="#">Super task button [optional]</a></p>
			</div>
		</div>
	</div>
</div>

<pre><code>&lt;div class="bg-center bg-cover" data-bgimg-srcset="img/1x1.png 991w, img/ip-cover-image-1200x726.jpg 992w">
	&lt;div class="container">
		&lt;div class="row mrgn-bttm-md">
			&lt;div class="col-md-7">
			&lt;h1 property="name" id="wb-cont">[Institution name]&lt;/h1>
				&lt;p>Short description of the institution’s mandate.&lt;/p>
				&lt;p>&lt;a class="btn btn-call-to-action" href="#">Super task button [optional]&lt;/a>&lt;/p>
			&lt;/div>
		&lt;/div>
	&lt;/div>
&lt;/div></code></pre>

```

Reply:
Added